client/src/components/Pages/BookingView/BookingInfo.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import moment from "moment";
+
+import { Row, Skeleton } from "antd";
+
+import { calculatePrice } from "./helpers";
+
+import {
+  BookingInfoWrapper,
+  InfoText,
+  InfoValue,
+  ErrorMsg
+} from "./PaymentsPlan.style";
+
+import {
+  SectionWrapperContent,
+  SectionTitle,
+  PayButton
+} from "../../Common/general";
+
+const BookingInfo = props => {
+  const { isLoading, data, handlePayNowClick } = props;
+  const {
+    bookingId,
+    price: fullPrice,
+    payedAmount,
+    startDate,
+    status,
+    endDate,
+    installments,
+    firstUnpaidInstallment,
+    couponDiscount,
+    coupons
+  } = data;
+
+  let discounts = 0;
+  if (!installments[0]) {
+    discounts = couponDiscount;
+  } else {
+    // get all coupons that have been used for this bookings
+    coupons.forEach(coupon => {
+      coupon.transactions &&
+        coupon.transactions.forEach(transaction => {
+          if (transaction.booking === bookingId) {
+            discounts += calculatePrice(15);
```

Comment:
Replace this hard coded value
```suggestion
            discounts += calculatePrice(transaction.usedDays);
```